### ISSUE TITLE
Would be nice to honor the `java.home` setting when launching dev mode instead of just $JAVA_HOME

### ISSUE THREAD

**Issue by scottkurz** [OpenLiberty/liberty-dev-vscode-ext#9]
The Maven VSCode extension has a setting:  `maven.terminal.useJavaHome` that will use the `java.home` setting to find Java, instead of only relying on the `JAVA_HOME` env var.  

That would be nice... it would allow more flexible switching between JDKs across envs for a single user.

**Comment by kathrynkodama** [comment]
The Maven VSCode extension also has a setting: `maven.terminal.customEnv` defined as follows:
```
Specifies an array of environment variable names and values. 
These environment variable values will be added to the terminal session before Maven is first executed.
```

Would it be helpful to also have a configurable setting like this in the Liberty Dev extension? 

FYI @patricktiu  @ericglau

**Comment by kathrynkodama** [comment]
Fixed with #12
